scapy/plist.py

```diff
@@ -211,11 +214,11 @@ def multiplot(self, f, lfilter=None, plot_xy=False, **kargs):
             kargs = MATPLOTLIB_DEFAULT_PLOT_KARGS
 
         if plot_xy:
-            lines = [plt.plot(*zip(*pl), **dict(kargs, label=k))
-                     for k, pl in d.iteritems()]
+            lines = [plt.plot(*list(zip(*pl)), **dict(kargs, label=k))
```

Comment:
I don't think so

```
In [1]: def toto(*args):
   ...:     print(args.__class__)
   ...:     

In [2]: toto(*zip([2]*4, [3]*3, [4]*5))
<class 'tuple'>
```

So the tuple should be recognize as *args for the plot function